pandas/core/arrays/numpy_.py

```diff
@@ -349,13 +351,19 @@ def all(self, axis=None, out=None, keepdims=False, skipna=True):
         nv.validate_all((), dict(out=out, keepdims=keepdims))
         return nanops.nanall(self._ndarray, axis=axis, skipna=skipna)
 
-    def min(self, axis=None, out=None, keepdims=False, skipna=True):
```

Comment:
What's the motivation for these changes? Pleasing mypy?

To the extent possible we'd like to keep the signature compatible with ndarray's methods, so that we work well with things like NumPy's dispatch APIs.

Reply:
Those keywords are still accepted as kwargs, and now actually are validated (`nv.validate_min`), which was before not the case. 
Those keywords don't do anything for this array class, so IMO it is good to put them in kwargs (we do the same pattern in eg Series.min).

Reply:
seems ok to me to allow kwargs; agree we do this elsewhere